packages/loader/container-loader/src/container.ts

```diff
@@ -1121,8 +1108,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private recordConnectStartTime() {
```

Comment:
can we move this function to connectionStateHandler

Reply:
New class now have core logic related to changing of states only.